src/screens/Login/Start/StartStyles.js

```diff
@@ -39,13 +39,11 @@ export default StyleSheet.create({
     marginBottom: 10,
     textAlign: 'center',
   },
-  primaryButton: {
-    marginBottom: 20,
-  },
   orText: {
     alignSelf: 'center',
     color: colors.textOnPurple,
     fontSize: 16,
+    marginVertical: 20,
```

Comment:
Not sure, that it is a better way (instead of primaryButton's marginBottom.
If we will delete 'or' text, then layout will be broken in this case. In previous variant it may be safely removed.

Reply:
At this case we just need to set one property to one element,instead of setting one margin property for both buttons around "or" text

Reply:
Yes, but as I described above, we will need to modify multiple elements in case of changes of 'or' text block.
From my point of view, each block should be easily removable with no side-effects to layout of other neighbour elements.